pypeit/spectrographs/magellan_fire.py

```diff
@@ -40,60 +118,51 @@ def __init__(self):
                             xgap            = 0.,
                             ygap            = 0.,
                             ysize           = 1.,
-                            platescale      = 0.15,
+                            platescale      = 0.18,
                             darkcurr        = 0.01,
-                            saturation      = 20000., # high gain mode, low gain is 32000
+                            saturation      = 20000., # high gain is 20000., low gain is 32000
                             nonlinear       = 1.0, # high gain mode, low gain is 0.875
                             numamplifiers   = 1,
                             gain            = 1.2, # high gain mode, low gain is 3.8 e-/DN
                             ronoise         = 5.0, # for high gain mode and SUTR read modes with exptime ~ 900s
-                            datasec         = '[1:2048,1:2048]',
-                            oscansec        = '[:,:4]'
+                            datasec         = '[5:2044,5:2044]',
```

Comment:
Is there a reason why you are trimming here?

Reply:
it has overscan region.